Makefile

```diff
@@ -51,10 +49,14 @@ lint:
 	flake8 rest_framework_bulk
 
 test:
-	python tests/manage.py test ${NOSE_FLAGS}
-
-test-coverage:
-	python tests/manage.py test ${NOSE_FLAGS} ${COVER_FLAGS}
+	python tests/manage.py test ${TEST_FLAGS}
+
+test-coverage: clean-test
+	-coverage run ${COVER_FLAGS} tests/manage.py test ${TEST_FLAGS}
```

Comment:
interesting. guess this change is because django-nose has issues with some newer django versions. any other reason?